test/api_compatibility.js

```diff
@@ -103,30 +96,78 @@ module.exports = function apiCompatibilityChecks(_createTracer) {
             });
         });
 
-        describe('SpanContext', function() {
-            var tracer = createTracer();
-            var span = tracer.startSpan('test_span');
-            var spanContext = span.context();
+        describe('Reference', function() {
+            it('should handle Spans and SpanContexts', function() {
+                expect(function() { new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, span); }).to.not.throw(Error);
+            });
+        });
+    });
 
-            it('should be a class', function() {
-                expect(spanContext).to.be.an('object');
+    describe('Opentracing implementation compatibility', function() {
+        var tracer;
+        var span;
+
+        beforeEach(function() {
+            tracer = createTracer();
+            span = tracer.startSpan('test-span');
+        });
+
+        describe('Tracer', function() {
+            _.each(spanFunctions, function(name) {
+                it(name + ' should be a method', function() {
+                    var span = tracer.startSpan('test-span');
+                    expect(span[name]).to.be.a('function');
```

Comment:
Its also asserting that whatever "startSpan" returns has span methods.  Its reusing the code above that validates spans.